### PR TITLE
USNP36 Establish Pull Request Template 

### PR DESCRIPTION
**Pull request by charlottekies**
### Summary of Changes

The changes in this PR include a new folder, .github, and a new file, GITHUB_PULL_REQUEST_TEMPLATE.md, that will be used for each Pull Request going forward. 

---

### Summary of Implementation Logic

Github automatically recognizes the .github/ directory as a configuration directory, and will implement the PR template in future pull requests made in this repo. According to ChatGPT. 

---

### Issue Link

Original Issue: https://github.com/RubySpeeders/US-Navy-Project/issues/36

Dependent Issues:

Depends on Issues:

---

### Screen shots / Videos

---

### Checklist for PR Owner

- [x] Standardized PR Title
- [ ] Included relevant videos and screen shots
- [ ] Included Summary of Changes
- [x] Included Summary of Implementation Logic

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Depends on Issues:
+
+---
+
+### Screen shots / Videos
+
+<!--- Place relevant screen shots and videos here that would help reviewers conceptualize the changes --->
+
+---
+
```
Comment (RubySpeeders) [inline .github/PULL_REQUEST_TEMPLATE.md:40]: Could we list one more thing: A/C (Acceptance Criteria). We should have this defined in the original issue and have a checklist. For example:
[ ] musician images are xxpx wide and xxpx tall
[ ] clicking on the button brings user to the correct page and /link
[ ] whatever the ticket is supposed to fix/bring into the code base

**Review by RubySpeeders — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (charlottekies) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: Love the checklist. OK, so we'll make a Checklist for AC in the original issue, and then port it over into the PR so the checklist exists in both places?

Comment (RubySpeeders) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: I think so! This will also force us (me...) to be more proactive when creating the original issue.

**Review by RubySpeeders — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (charlottekies) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: Actually happened to me with the backend ticket--I forgot I included adding photos to the backend as an AC

Comment (charlottekies) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: !OK added the section

**Pull request merged**